packages/graph/hash_graph/lib/graph/src/api/rest/utoipa_typedef/subgraph/vertices/vertex.rs

```diff
@@ -9,12 +9,31 @@ use crate::{
 #[derive(Debug, PartialEq, Eq, Serialize)]
 #[serde(tag = "kind", content = "inner")]
 #[serde(rename_all = "camelCase")]
+#[expect(clippy::enum_variant_names)]
```

Comment:
I don't really understand how we keep removing and re-adding this? Is something confusing clippy

Reply:
Cippy will only complain about these kinds of things if the struct is not exposed. It, however, cannot know, that we are exposing that via the OpenAPI specs.